var/spack/repos/jcsda-emc-bundles/packages/nceplibs-bundle/package.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Comment:
If all other packages in this directory are called `...-env`, should we rename the `nceplibs-bundle` (or all others) for consistency?

Reply:
Yes, same thought about the name.